### ISSUE TITLE
[Feature]: Research conda nightly builds for latest `main` 

### ISSUE THREAD

**Issue by tomvothecoder** [xCDAT/xcdat#265]
### Is your feature request related to a problem?

A nightly build might be helpful for users/developers who want to test/use the latest version of `xcdat` without having to setup the development environment in this repo.

### Describe the solution you'd like

One possible way might be to have a GH Actions job that automatically builds a “latest” version of the package and pushes it to Anaconda whenever there is a new commit on `main`.

### Additional context

@pochedls brought this idea up in our Slack exchange

**Comment by lee1043** [comment]
While I think having nightly is convenient for using latest build without manual install in the dev env, but for PMP, I have retracted it when we move PMP to conda-forge channel. I think one of the reasons discussed was that with nightly version  becomes not very traceable while reproducibility is important for PMP. As PMP is transitioning to build on xCDAT, version tracking of xCDAT would be important to PMP. I think I would prefer the standard way of version releasing, and if we setup nightly release in addition, I think we might want to keep it internally and not advertise too widely. Just my 2 cents.

**Comment by pochedls** [comment]
Maybe this is a discussion item..? I am in the pro-nightly build camp, but agree that we could and should hide it so that a new user doesn't accidentally install it as their default xcdat instance.

**Comment by rljacob** [comment]
Yes dependent packages should use tagged versions, not latest head-of-main.
